src/structures/ApplicationCommand.ts

```diff
@@ -0,0 +1,39 @@
+import BulbBotClient from "./BulbBotClient";
+import { CommandInteraction, Permissions, PermissionString } from "discord.js";
+import { ApplicationCommandOptions, ApplicationCommandType } from "../utils/types/ApplicationCommands";
+
+export default class ApplicationCommand {
+	public readonly client: BulbBotClient;
+	public readonly type: ApplicationCommandType;
+	public readonly name: string;
+	public readonly description: string;
+	public readonly dm_permissions: boolean;
+	public readonly default_member_permissions: string | null;
+	public readonly command_permissions: PermissionString[];
+	public readonly options: ApplicationCommandOptions[];
+
+	constructor(client: BulbBotClient, options: any) {
```

Comment:
Typedef for `options` please. Is this supposed to be `ApplicationCommandOptions` from `src/utils/types/ApplicationCommands.ts`? If so, what is `options.options` (line 23)?

Reply:
Already fixed this one of my local builds, would also like to just deconstruct the entire options object because options.options sounds werid